.gitlab/source_test/macos.yml

```diff
@@ -67,6 +67,7 @@ tests_macos:
   script:
     - !reference [.retrieve_linux_go_deps]
     - !reference [.retrieve_linux_go_tools_deps]
+    - inv -e install-tools
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Would it be better to move this to `.macos_gitlab` within the `before_script` section? As it is the same as `.tests_macos_gitlab` and only these two jobs are extending from this job IIUC